react/css/src/digitv2/components/fileUploadV2.scss

```
@@ -4,12 +4,12 @@
 .digit-upload-image-drawer {
   @apply w-full fixed left-0 right-0 flex-col;
   display: flex;
-  background-color: theme("digitv2.lightTheme.paper-primary");
-  border-radius: theme("digitv2.spacers.spacer2") theme("digitv2.spacers.spacer2") theme("digitv2.spacers.spacer0") theme("digitv2.spacers.spacer0");
+  background-color: var(--digitv2-lightTheme-paper-primary);
+  border-radius: var(--digitv2-spacers-spacer2) var(--digitv2-spacers-spacer2) var(--digitv2-spacers-spacer0) var(--digitv2-spacers-spacer0);
```

Stylelint (17.3.0)

[error] Unexpected duplicate "border-radius" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)